From 96e76ee482dad2a0d71f9a5a5a6558d272d538ca Mon Sep 17 00:00:00 2001
From: Antoine Jacoutot <ajacoutot@gnome.org>
Date: Sun, 24 Oct 2021 10:26:03 +0200
Subject: [PATCH] Unbreak build without systemd.

This fixes the check for WITH_SYSTEMD and prevents a fatal error:
'systemd/sd-daemon.h' file not found when building with systemd disabled.
---
 gcr/gcr-ssh-agent-service.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcr/gcr-ssh-agent-service.c b/gcr/gcr-ssh-agent-service.c
index 3b5a4ca..b7e8928 100644
--- a/gcr/gcr-ssh-agent-service.c
+++ b/gcr/gcr-ssh-agent-service.c
@@ -39,7 +39,7 @@
 #include <glib/gstdio.h>
 #include <gcr/gcr-base.h>
 
-#ifdef WITH_SYSTEMD
+#if WITH_SYSTEMD
 #include <systemd/sd-daemon.h>
 #endif
